daux/docs/900_Distributed_Events/300_Event_Listeners_for_Clients.md

```diff
@@ -9,6 +9,6 @@ You can add event listeners to a Hazelcast Java client. You can configure the fo
 <br></br>
 ***RELATED INFORMATION***
 
-*Please refer to the [Client Listenerconfig section](/1600_Hazelcast_Clients/100_Java_Client/300_Configuration/700_Listeners.md) for more information.*
+*Please refer to the [Configuring Client Listeners section](/Hazelcast_Clients/Java_Client/Configuration/Listeners.md) for more information.*
```

Comment:
Why are the number prefixes removed in the link?

I don't fully understand the processing, but it's suspicious to me, that some links use the prefixes and some other does not use them.

Reply:
Actually, daux doesn't care about the numbers. The HTML generation process removes all the numbers in the links.